common/build/eslint-plugin-fluid/package.json

```diff
@@ -0,0 +1,39 @@
+{
+	"name": "@fluid-internal/eslint-plugin-fluid",
+	"version": "0.1.0",
+	"private": true,
+	"description": "Custom ESLint rules for the Fluid Framework",
+	"homepage": "https://fluidframework.com",
+	"repository": {
+		"type": "git",
+		"url": "https://github.com/microsoft/FluidFramework.git",
+		"directory": "common/build/eslint-pluig-fluid/"
+	},
+	"license": "MIT",
+	"author": "Microsoft and contributors",
+	"main": "index.js",
+	"scripts": {
+		"build": "npm run build:readme",
+		"build:readme": "markdown-magic --files \"**/*.md\"",
+		"clean": "rimraf --glob nyc",
+		"format": "npm run prettier:fix",
+		"prettier": "prettier --check . --cache --ignore-path ../../../.prettierignore",
+		"prettier:fix": "prettier --write . --cache --ignore-path ../../../.prettierignore",
+		"test": "mocha src/test"
+	},
+	"dependencies": {
+		"@microsoft/tsdoc": "^0.14.2",
```

Comment:
Be sure to do an audit of the dependencies in `eslint-config-fluid` - now that some of the code has been moved out of that package, some of these deps can probably be removed from it.

Reply:
Good point. Removed some dependencies and devDependencies from the package.